src/style.css

```diff
@@ -0,0 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.product {
+  display: flex;
+}
+
+.product__card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: auto;
+}
+
+.product__image {
+  width: 100%;
+  margin: 32px auto 0;
+}
+
+.product__image--image {
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.product__container {
+  margin: 40px 16px 16px;
+}
+
+.product__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0;
+  padding-bottom: 4px;
+}
+
+.product__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0;
+  padding-bottom: 16px;
+}
+
+.product__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.product__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  margin: 0;
+}
+
+.product__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 24px 0 16px;
+}
+
+.product__price_price {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+  margin: 0;
+}
+
+.product__price_value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0;
+}
+
+.product__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 38px;
+  background: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.product__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
Change only border-color to blue. See more details in checklist, point 9

Reply:
it's a general rule that has border-style and border-color